includes/controls/media.php

```diff
@@ -47,6 +47,7 @@ public function get_default_value() {
 		return [
 			'url' => '',
 			'id' => '',
+			'size' => 'full',
```

Comment:
do we want it to be saved in the DB by default?
maybe is should be calculated as default value on render?

Reply:
Full should be the default when the user is not selecting size. The original image size.

Reply:
yes, but the user might have unused data in the DB. We need to ensure that this value isn't saved when not needed

Reply:
Fixed, as soon as you select a desired size and save, only this is saved in the db, that is, a controller that does not have the selection will not populate the database with the default value (`full`).